### PR TITLE
DelegateGroups now have a DegreeType as a possible attribute FIST-87 #resolve

### PR DESCRIPTION
**Pull request by gil-l**
This allows for groups encompassing all delegates for degrees of said type.

### CODE REVIEW

File: fenixedu-ist-delegates/src/main/java/pt/ist/fenixedu/delegates/domain/accessControl/DelegateGroup.java [hunk 1]
```diff
@@ -42,22 +42,27 @@
 public class DelegateGroup extends CustomGroup {
     private static final long serialVersionUID = -7261567414029131275L;
 
-    @GroupArgument("")
+    @GroupArgument
```
Comment (pedrosan7os) [inline fenixedu-ist-delegates/src/main/java/pt/ist/fenixedu/delegates/domain/accessControl/DelegateGroup.java]: Why this change?

**Comment by gil-l** [comment]
hmmm, in the traditional portuguese way, I have no memory of this.... probably mindlessly copied and pasted or made to be the same as some other group. I'll fix it in a bit. Should I also use "" on the DegreeType so it doesn't require the key in the group language?

**Comment by pedrosan7os** [comment]
Use "" for the most commonly used field, since it compresses the expression syntax for those cases, you can only use in one.
